src/theme/themeProvider.stories.mdx

```diff
@@ -55,6 +64,808 @@ Custom palette colours can be specified. For example, the primary, secondary or
+                <C.Tabs
+                  value={value}
+                  onChange={handleChange}
+                  aria-label="simple tabs example"
+                >
+                  <C.Tab label="Item One" />
+                  <C.Tab label="Item Two" />
+                  <C.Tab label="Item Three" />
+                </C.Tabs>
+              </C.AppBar>
+              <C.TabPanel p={3} selected={value} index={0}>
+                Item One
+              </C.TabPanel>
+              <C.TabPanel p={3} selected={value} index={1}>
+                Item Two
+              </C.TabPanel>
+              <C.TabPanel p={3} selected={value} index={2}>
+                Item Three
+              </C.TabPanel>
+            </C.Box>
+          )
+        })}
+      </C.Flex>
+      <C.Caption>
+        <C.Link href="/components/?path=/docs/design-system-typography--typography">
+          Typography
+        </C.Link>
+      </C.Caption>
+      <C.Flex
+        flexGrow={1}
+        p={3}
+        pt={4}
+        m={1}
+        border="1px solid black"
+        bgcolor="background.default"
+      >
+        <C.Box width={1 / 2}>
+          <C.Typography>System Typography</C.Typography>
+          <C.Heading.h2 gutterBottom>h2. Heading</C.Heading.h2>
+          <C.Heading.h3 gutterBottom>h3. Heading</C.Heading.h3>
+          <C.Heading.h4 gutterBottom>h4. Heading</C.Heading.h4>
+          <C.Subheading.h3 gutterBottom>
+            subheading. Lorem ipsum dolor sit amet, consectetur adipisicing
+            elit. Quos blanditiis tenetur
+          </C.Subheading.h3>
+          <C.Typography variant="body1" gutterBottom>
+            body1. Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+            Quos blanditiis tenetur unde suscipit, quam beatae rerum inventore
+            consectetur.
+          </C.Typography>
+          <C.Typography variant="body2" gutterBottom>
+            body2. Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+            Quos blanditiis tenetur unde suscipit, quam beatae rerum inventore
+            consectetur.
+          </C.Typography>
+        </C.Box>
+        <C.Box width={1 / 2}>
+          <C.Text>Disply Typography</C.Text>
+          <C.Display.d1 gutterBottom>d1. Heading</C.Display.d1>
+          <C.Display.d2 gutterBottom>d2. Heading</C.Display.d2>
+          <C.Text gutterBottom>
+            Text. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quos
+            blanditiis tenetur unde suscipit, quam beatae rerum inventore
+            consectetur.
+          </C.Text>
+          <C.Monospace wrap>
+            Monospace. Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+            Quos blanditiis tenetur unde suscipit, quam beatae rerum inventore
+            consectetur.
+          </C.Monospace>
+        </C.Box>
+      </C.Flex>
+    </Background>
+  </C.ThemeProvider>
+</Story>
+</Preview>
+
 ## Props
 
 <Props of={ThemeProviderDocs} />
```

Comment:
How do all the new possible props look here?
If not shown in this that may need documenting.

Reply:
I've added all of the new props to the story now.